include a description of WORKFLOW_SHARE_DIR/bin and lib/python (#609)
cylc/cylc-doc: src/user-guide/installing-workflows.rst

   │       └── <time-stamp>-reinstall.log
   └── .service

.. note::

   ``bin`` and ``lib/python`` directories in the :ref:` workflow share directory <WorkflowShareDirectories>`.

> 

Reviewer: Something got cut

Author: Fixed.

   │       └── <time-stamp>-reinstall.log
   └── .service

.. note::

   If your workflow needs to create or install scripts or executables at runtime
   and you don't want Cylc to delete them on re-installation, you can use

>    equivalent directories in the :ref:`WorkflowShareDirectories`.

Reviewer: What do you mean equivalent directories? There seems to be missing context here

Author: ```suggestion
   ``bin`` and ``lib/python`` directories in the :ref:` workflow share directory <WorkflowShareDirectories>`.
```
